packages/metrics/tests/unit/Metrics.test.ts

```diff
@@ -747,4 +798,44 @@ describe('Class: Metrics', () => {
       expect(loggedData._aws.CloudWatchMetrics[0].Namespace).toEqual(namespace);
     });
   });
+
+  describe('concept', ()=>{
```

Comment:
@dreamorosi should the test for the concept type `MetricResolution` as const be removed , does it fulfilled its purpose ?

Reply:
Yes it did thank you! I think it can be removed.